src/services/user/services/AdminUsers.js

```diff
@@ -97,6 +101,10 @@ class AdminUsers {
 		}
 	}
 
+	formatBirthdayOfUsers(users) {
+		users.forEach((user) => { user.birthday = moment(user.birthday).format('DD.MM.YYYY'); });
```

Comment:
Please provide one or two tests for this (service level + unit tests)

Reply:
done